src/scss/components/checkbox/mixins/_hxCheckboxControl.scss

```diff
@@ -0,0 +1,112 @@
+@import "vars";
+@import "hxCheckboxFacade";
+
+//
+// ===== Mixins for <hx-checkbox-control> states =====
+//
+
+// ========== PRISTINE ==========
+@mixin __checkboxControl($psuedo-state: null) {
+  @if $psuedo-state == focus {
+    box-shadow: $focus-glow;
+
+    &::-ms-check {
+      border-color: $cyan-700;
+    }
+
+    ~ label > hx-checkbox {
+      border-color: $cyan-700;
+      box-shadow: $focus-glow;
+    }
+  }
+  @else if $psuedo-state == hover {
+    color: $cyan-500;
+
+    &::-ms-check {
+      background-color: $cyan-50;
+      color: $cyan-500;
+    }
+  }
+}
+
+// ========== INVALID ==========
+@mixin __checkboxControl--invalid($psuedo-state: null) {
+  color: $red-900;
+
+  &::-ms-check {
+    border-width: 2px;
+    border-color: $red-900 !important;
+    color: $red-900 !important;
+  }
+
+  @if $psuedo-state == focus {
+    box-shadow: $focus-glow-invalid;
+
+    &::-ms-check {
+      border-color: $red-900;
+    }
+
+    ~ label > hx-checkbox {
+      border-color: $red-900;
+      box-shadow: $focus-glow-invalid;
+      border-width: 2px;
+    }
+  }
+  @else if $psuedo-state == hover {
+    &::-ms-check {
+      background-color: $red-100;
+    }
+
+    ~ label > hx-checkbox {
+      border: 2px solid $red-900;
+      color: $red-900;
+
+      &:hover {
+        background-color: $red-100;
+        border-color: $red-900;
+        color: $red-900;
+      }
+    }
+  }
+}
+
+// ========== DISABLED ==========
+@mixin __checkboxControl--disabled($psuedo-state: null) {
+  color: $gray-500;
+  cursor: not-allowed;
+
+  &::-ms-check {
+    background-color: $gray-100 !important; // simplest way of styling
+    border-width: 1px;
+  }
+
+  @if $psuedo-state == focus {
+    &::-ms-check {
+      border-color: $gray-500;
+      box-shadow: $focus-glow-disabled;
+    }
+
+    ~ label > hx-checkbox {
+      @include hxCheckboxFacade(disabled, focus);
+    }
+  }
+  @else if $psuedo-state == hover {
+    &::-ms-check {
+      background-color: $gray-100;
+      border-color: $gray-500;
+      color: $gray-500;
+    }
+  }
+}
+
+@mixin hxCheckboxControl($state: null, $psuedo-state: null) {
```

Comment:
How would I do `:focus:hover` styling?

Maybe change the arity to the following...
```scss
@mixin hxCheckboxControl($state: null, $hovered: false, $focused: false) { ... }
```

With this change, you can pass in `$hovered: true`, `$focused: true`, both, or neither.  All named parameters in SCSS mixins can be changed via keyword _or_ position. 

So...

```scss
/* all of these are equivalent */
@include hxCheckboxControl($hovered: true, $focused: false, $state: null)
@include hxCheckboxControl($focused: false, $hovered: true)
@include hxCheckboxControl($hovered: true, $focused: false)
@include hxCheckboxControl($hovered: true)
```